price-estimator/README.md

```diff
@@ -2,22 +2,27 @@
 
 The api is documented with [OpenAPI](https://www.openapis.org/) in `openapi.yml`. It can be accessed in a rendered and interactive fashion on https://editor.swagger.io/ through `File -> Import file`.
 
-TODO: Add a binary or docker image that can host the interactive ui without an external service.
+Alternatively locally with docker
+
+```
+sudo docker run -p 80:8080 -e SWAGGER_JSON=/openapi.yml --mount type=bind,src="$PWD/openapi.yml",dst=/openapi.yml swaggerapi/swagger-ui
```

Comment:
`sudo` is only necessary if you aren't in the `docker` (or something like that) group. I would also suggest removing it.